### PR TITLE
New event capture

### LINKED ISSUE #3: Code the logic in the server to receive the data and save it on the non-relational database
- [x] Save a JSON body in the Mongo
- [x] Validate if the "event" key is not missing
- [x] Add auth
- [x] Update documentation
- [x] If the field "timestamp" is not in the JSON body from the request, put the default now
- [x] Re-check the validations(min_length, max_length)
- [x] Get user and pass from env
- [x] Add init file

Add env vars in the Puppet:
- [x] EVENT_CAPTURE_USERNAME
- [x] EVENT_CAPTURE_PASSWORD
- [x] EVENT_CAPTURE_MONGO_URI

### PR DESCRIPTION
**Pull request by Marcelo-Theodoro**
Closes  #3

### CODE REVIEW

File: event-capture/auth.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from eve.auth import BasicAuth
+
+
+class Auth(BasicAuth):
```
Comment (seocam) [inline event-capture/auth.py]: Basic Auth it's the actual HTTP basic auth?

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,11 @@
-Flask
-mongo
-gunicorn
+Cerberus==1.2
+click==6.7
+Eve==0.8
+Events==0.3
+Flask==1.0.2
+itsdangerous==0.24
```
Comment (seocam) [inline requirements.txt:6]: waaaat?! lol

Comment (Marcelo-Theodoro) [reply requirements.txt:6]: haha, a bad name.
But it's just a Flask requirement.

**Review by muriloviana — APPROVED** [review]

File: requirements.txt (hunk 2 above)
Comment (mawkee) [reply requirements.txt:6]: Should we do the whole requirements.in/requirements.txt? I like that idea

**Review by mawkee — APPROVED** [review]

**Pull request merged**